backstop/tests/header.html

```diff
@@ -966,17 +967,24 @@ <h2>Split Menu Buttons</h2>
 
           <!-- Project Button -->
           <nav aria-label="breadcrumbs">
-            <button
-              class="iui-button iui-borderless iui-header-button iui-large"
-              type="button"
-              aria-label="Project picker"
-              aria-expanded="false"
-            >
-              <span class="iui-label">
-                <div class="iui-name">Project Alpha</div>
-              </span>
+            <span class="iui-button-split-menu">
+              <button
+                class="iui-button  iui-borderless iui-header-button iui-large"
```

Comment:
Extra space between iui-button and iui-borderless.

Weird it didn't auto format

Reply:
Removed extra space